app/src/main/java/com/asfoundation/wallet/referrals/ReferralModel.kt

```diff
@@ -3,7 +3,10 @@ package com.asfoundation.wallet.referrals
 import java.math.BigDecimal
 
 data class ReferralModel(
```

Comment:
Please create a secondary constructor for this purpose:

`package com.asfoundation.wallet.referrals

import java.math.BigDecimal

data class ReferralModel constructor(
    val completed: Int, val link: String?, val invited: Boolean, val pendingAmount: BigDecimal,
    val amount: BigDecimal, val symbol: String, val maxAmount: BigDecimal,
    val minAmount: BigDecimal, val available: Int, val receivedAmount: BigDecimal,
    val isRedeemed: Boolean, val isActive: Boolean) {

  constructor() : this(0, "", false, BigDecimal.ZERO, BigDecimal.ZERO,
      "", BigDecimal.ZERO, BigDecimal.ZERO, 0, BigDecimal.ZERO, false,
      false)
}`